internal/consensus/state_test.go

```diff
@@ -705,31 +716,39 @@ func TestStateLockPOLRelock(t *testing.T) {
 	ensureNewRound(newRoundCh, height+1, 0)
 }
 
-// 4 vals, one precommits, other 3 polka at next round, so we unlock and precomit the polka
-func TestStateLockPOLUnlock(t *testing.T) {
+// TestStatePOLDoesNotUnlock tests that a validator maintains its locked block
+// despite receiving +2/3 nil prevotes and nil precommits from other validators.
```

Comment:
Receiving of +2/3 nil precommits was never the condition to unlock. So maybe it would be good removing it from the comment.

Reply:
As stated above, will do here.